tools/gethnet/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ethereum/client-go:stable
+
+WORKDIR /gethnet
+COPY datadir datadir/
+COPY config.toml .
+
+ENTRYPOINT [ "geth" ]
+CMD [\
+"--dev",\
+"--datadir", "/gethnet/datadir", \
+"--mine", \
```

Comment:
Does this mean mining is enabled?  Is this used only for testing?

Reply:
for testing or for development, it mines blocks every 2 seconds